### ISSUE TITLE
Add table to prevent daemon access on every URL resolve

### ISSUE THREAD

**Issue by kauffj** [lbryio/spee.ch#129]
We should have a data structure that allows us to re-resolve already resolved URLs without asking the daemon every time.

**Comment by billbitt** [comment]
I made these changes in the claim-table branch.  I have spee.ch running off of this branch right now to test it out.  So far so good, and it is much faster to serve content since it does not need to ping the daemon for info unless it is a brand new claim.

I used a version of @filipnyquist 's lighthouse sync tool to load all free claims into a sql table.  That table holds the most up to date `resolve` data for every name/claimId pair.  The tool will continue to monitor lbrycrdd and then resolve claims and upsert them into the db on each new block.  Spee.ch can use that table to get claim lists and resolve info, and only needs to use the lbrynet daemon for `get` and `publish` requests.  

Potential bugs/holes in the current code: if a claim is updated, the newest resolved data will go into the Claim table, but spee.ch will see the record in the File table first and think it already has the most up to date claim.  I need to add a loop to check the File record's outpoint against the outpoint in the Claim table, and update the claim with `get` if needed.
